src/drivers/wa-sqlite/database.ts

```diff
@@ -106,7 +106,11 @@ export class ElectricDatabase implements Database {
 
     // Open the DB connection
     // see: https://rhashimoto.github.io/wa-sqlite/docs/interfaces/SQLiteAPI.html#open_v2
-    const db = await sqlite3.open_v2(dbName)
+    const db = await sqlite3.open_v2(
+      dbName,
+      SQLite.SQLITE_OPEN_CREATE | SQLite.SQLITE_OPEN_READWRITE,
```

Comment:
I assume this is right but just to be 💯 providing both flags with a `|` means “create or open”?

Reply:
This specific combination is described here: https://sqlite.org/c3ref/open.html

> SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE
>     The database is opened for reading and writing, and is created if it does not already exist. This is the behavior that is always used for sqlite3_open() and sqlite3_open16().